Command/RunParatestCommand.php

```diff
@@ -43,12 +44,17 @@ protected function prepare()
         $cleanProcess = new Process("rm -fr $this->testDbPath/dbTest.db $this->testDbPath/dbTest*.db*");
         $cleanProcess->run();
         $this->output->writeln("Creating Schema in $this->testDbPath ...");
-        $createProcess = new Process('php app/console doctrine:schema:create --env=test');
-        $createProcess->run();
+        $command = $this->getApplication()->find('doctrine:schema:create');
+        $input = new ArrayInput(['--env' => 'test']);
```

Comment:
IIRC, the Bundle is still supporting Symfony 2.3 so you should probably use the old `array()` syntax.